packages/eslint-plugin-edu-react/package.json

```diff
@@ -1,23 +1,23 @@
 {
-  "name": "@chanzuckerberg/eslint-config-edu-react",
+  "name": "@chanzuckerberg/eslint-plugin-edu-react",
```

Comment:
To add a custom rule I had to convert this from a "config" to a "plugin" (eslint is kind of weird, and "configs" are just config, but "plugins" can have config and new rules).

An alternative would be to separate into both a config and a plugin, but that seems like too much work and not enough benefit to me.